lib/bitstyles_phoenix/component/dropdown.ex

```diff
@@ -478,7 +514,16 @@ defmodule BitstylesPhoenix.Component.Dropdown do
     extra =
       assigns
       |> assigns_to_attributes()
-      |> Keyword.put_new(:variant, "menu")
+
+    extra =
+      if Bitstyles.version() >= "5.0.0" do
```

Comment:
We really have to start refactoring version comparisons to not use strings at some point 🙈 
But good for now.

Reply:
Yes 😇 I'll try to squeeze in that work in my schedule after 5.0.0 and 6.0.0 upgrades are merged